static/scripts/index.js

```diff
@@ -0,0 +1,513 @@
+fetch('/assignment/load_data')
+    .then(response => response.json())
+    .then(loadData => {
+        console.log(loadData);
+
+        const {courses, users, teachers, researchers, preferences, organizations, current_year} = loadData;
+        const example = document.getElementById("handsontable");
```

Comment:
`example` is not the ideal constant name. However, you can simply put the selector where the constant is used.